windows/src/developer/TIKE/child/UfrmPackageEditor.dfm

```diff
@@ -1032,78 +1075,215 @@ inherited frmPackageEditor: TfrmPackageEditor
           TabOrder = 0
           OnClick = cmdBuildPackageClick
         end
-        object cmdInstall: TButton
-          Left = 15
-          Top = 204
-          Width = 133
-          Height = 25
-          Caption = '&Install Package...'
-          TabOrder = 1
-          OnClick = cmdInstallClick
-        end
         object editOutPath: TEdit
-          Left = 168
-          Top = 82
-          Width = 349
+          Left = 111
+          Top = 118
+          Width = 383
           Height = 21
           TabStop = False
           ParentColor = True
           ReadOnly = True
-          TabOrder = 2
+          TabOrder = 4
         end
         object cmdOpenContainingFolder2: TButton
-          Left = 15
-          Top = 344
+          Left = 293
+          Top = 80
           Width = 133
           Height = 25
           Caption = '&Open Containing Folder'
-          TabOrder = 3
+          TabOrder = 2
           OnClick = cmdOpenContainingFolder2Click
         end
         object cmdAddToProject: TButton
-          Left = 15
-          Top = 376
+          Left = 432
+          Top = 80
           Width = 133
           Height = 25
           Action = modActionsMain.actProjectAddCurrentEditorFile
-          TabOrder = 4
+          TabOrder = 3
         end
-        object cmdUninstall: TButton
-          Left = 15
-          Top = 236
+        object cmdCompileInstaller: TButton
+          Left = 154
+          Top = 80
           Width = 133
           Height = 25
-          Caption = '&Uninstall Package'
+          Caption = 'Compile I&nstaller'
+          TabOrder = 1
+          OnClick = cmdCompileInstallerClick
+        end
+        object panBuildMobile: TPanel
+          Left = 328
+          Top = 162
+          Width = 310
+          Height = 265
+          BevelOuter = bvNone
+          Color = 15921906
+          ParentBackground = False
           TabOrder = 5
-          OnClick = cmdUninstallClick
+          object lblDebugHostCaption: TLabel
+            Left = 12
+            Top = 70
+            Width = 252
+            Height = 13
+            Caption = 'Debug host is active at following web addresses:'
+          end
+          object lblCrossPlatform: TLabel
+            Left = 12
+            Top = 6
+            Width = 169
+            Height = 17
+            Caption = 'Android and iOS Targets'
+            Font.Charset = DEFAULT_CHARSET
+            Font.Color = clWindowText
+            Font.Height = -14
```

Comment:
Is `-14` a valid font height? I see three of them on this PR

Reply:
Yes, this is a Windows anachronism. https://support.microsoft.com/en-us/help/74299/info-calculating-the-logical-height-and-point-size-of-a-font